src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { registerColor } from 'vs/platform/theme/common/colorRegistry';
+import { Color, RGBA } from 'vs/base/common/color';
+import * as nls from 'vs/nls';
+
+
+// ----- text colors
+
+export const textLinkActiveForeground = registerColor('textLink.activeForeground', { light: '#006AB1', dark: '#3794FF', hc: '#3794FF' }, nls.localize('textLinkActiveForeground', "Foreground color for links in text when clicked on and on mouse hover."));
```

Comment:
You should really try to use existing colors for these sort of things. Again, extensions have to opt into supplying these colors, so using existing colors will mean themes will work properly for them. If you define new colors, you won't get proper theme behavior.

Reply:
I've moved over everything I can for the color registry stuff. The only things that remain in the new registry are a secondary button, disabled button, and some gradients, and drop-shadows. None of the those or comparable elements exist in the current registry.

Reply:
I've tested on the themes, and they look good.

Reply:
![image](https://user-images.githubusercontent.com/60623315/78070706-20ea6980-7351-11ea-83dc-3004f7028a3b.png)

Reply:
![image](https://user-images.githubusercontent.com/60623315/78070748-2e9fef00-7351-11ea-8d82-0e284b36d4e4.png)